ui/app/components/page/userpass-reset-password.hbs

```diff
@@ -0,0 +1,54 @@
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 data-test-title class="title is-3">
+      Reset password
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+{{#if this.successful}}
+  <div class="box is-sideless is-fullwidth is-marginless">
+    <Hds::Alert @type="inline" @color="success" data-test-reset-password-success as |A|>
+      <A.Title>Password set successfully</A.Title>
+      <A.Button @text="Reset form" @color="secondary" {{on "click" this.reset}} data-test-reset-reset-password />
```

Comment:
Thinking about reset password forms I've used -  I'd expect an alert to render that it was successful above the form. It feels like we don't need to include a `Reset form` button + action to conditionally render the form.

But perhaps you had other reasons for doing so! Just a small comment about the UX here